### PR TITLE
Add casio screen receiver 3.02b

### PR DESCRIPTION
**Pull request by numbermaniac**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no open pull requests for the same cask.
- [x] Checked that the cask was not already refused in closed issues.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md

----

Version and URL need to be fixed, see the comment on the URL line; also, let me know if I used the wrong name.

On the homepage, you have to check the box that says you accept the terms and conditions before downloading. What do I need to do for this?

### CODE REVIEW

**Comment by adidalal** [comment]
Re: T&C, there's an open issue somewhere about how to handle licenses but it should be OK for now. If the accept sets a cookie or something to allow download you'll need to add it in as a URL parameter (i.e. Like java)

**Comment by Numbermaniac** [comment]
Thanks. No cookie is required in this case, there's simply a checkbox at the top, which keeps all the download boxes faded out until you check it.

File: Casks/casio-screen-receiver.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask 'casio-screen-receiver' do
+  version '3.02_b,4'
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: Please change this to `version '302,b:4'`.

File: Casks/casio-screen-receiver.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  sha256 '5eaff7fd7894d483534d82e7fbc6b77a6d304afa580e10b1c7bce38a3a69e07a'
+
+  url "https://edu.casio.com/education/support_software/dl/screen_Receiver/screenrecv_inst_302b_#{version.after_comma}.zip" # 302b should be interpolated from the version
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: Please change this to `url "https://edu.casio.com/education/support_software/dl/screen_Receiver/screenrecv_inst_#{version.before_comma}#{version.after_comma_befor_colon}_#{version.after_colon}.zip"`.

File: Casks/casio-screen-receiver.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  name 'Casio Screen Receiver'
+  homepage 'https://edu.casio.com/education/support_software/'
+
+  pkg "ScreenReceiver_Ver.#{version.before_comma}.pkg"
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: Please change this to `pkg "ScreenReceiver_Ver.#{version.before_comma}_#{version.after_comma_befor_colon}.pkg"`.

Comment (Numbermaniac) [inline Casks/casio-screen-receiver.rb]: This would come out to 302b though, it's meant to be 3.02_b.

Comment (miccal) [inline Casks/casio-screen-receiver.rb]: I just updated my comments, so it should work now.

Comment (Numbermaniac) [inline Casks/casio-screen-receiver.rb]: All updated, thanks!

File: Casks/casio-screen-receiver.rb [hunk 4]
```diff
@@ -1,12 +1,12 @@
 cask 'casio-screen-receiver' do
-  version '3.02_b,4'
+  version '3.02,b:4'
   sha256 '5eaff7fd7894d483534d82e7fbc6b77a6d304afa580e10b1c7bce38a3a69e07a'
 
-  url "https://edu.casio.com/education/support_software/dl/screen_Receiver/screenrecv_inst_302b_#{version.after_comma}.zip" # 302b should be interpolated from the version
+  url "https://edu.casio.com/education/support_software/dl/screen_Receiver/screenrecv_inst_#{version.before_comma_no_dots}#{version.after_comma_before_colon}_#{version.after_colon}.zip" # 302b should be interpolated from the version
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: You will need to remove your comment.

File: Casks/casio-screen-receiver.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+cask 'casio-screen-receiver' do
+  version '3.02,b:4'
+  sha256 '5eaff7fd7894d483534d82e7fbc6b77a6d304afa580e10b1c7bce38a3a69e07a'
+
+  url "https://edu.casio.com/education/support_software/dl/screen_Receiver/screenrecv_inst_#{version.before_comma_no_dots}#{version.after_comma_before_colon}_#{version.after_colon}.zip" # 302b should be interpolated from the version
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: Apologies, it should be `#{version.before_comma.no_dots}` not `#{version.before_comma_no_dots}`.

File: Casks/casio-screen-receiver.rb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  name 'Casio Screen Receiver'
+  homepage 'https://edu.casio.com/education/support_software/'
+
+  pkg "ScreenReceiver_Ver.#{version.before_comma}_#{version.after_comma_before_colon}.pkg"
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: Same here.

File: Casks/casio-screen-receiver.rb (hunk 4 above)
Comment (Numbermaniac) [inline Casks/casio-screen-receiver.rb]: Whoops, forgot. All done.

File: Casks/casio-screen-receiver.rb [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+  pkg "ScreenReceiver_Ver.#{version.before_comma}_#{version.after_comma.before_colon}.pkg"
+
+  uninstall pkgutil: [
+                       'com.casio.fx.ScreenReceiver.ScreenReceiver',
+                       'com.casio.fx.ScreenReceiver.fxASPIDriver',
+                       'com.casio.fx.ScreenReceiver.fxSRManual',
+                     ],
+            delete:  "#{appdir}/CASIO/ScreenReceiver.app",
+            rmdir:   "#{appdir}/CASIO"
```
Comment (miccal) [inline Casks/casio-screen-receiver.rb]: One last thing - `#{appdir}` won’t be respected because it’s a `pkg`, so please change these to `/Applications`.

**Comment by Numbermaniac** [comment]
Not a problem @miccal.

**Pull request merged**